core/firebaseRestAPI.js

```diff
@@ -72,7 +72,7 @@ function ensureDefaultsForOrganization(organization) {
   return R.merge(ORGANIZATION_SCHEMA, camelize(organization))
 }
 
-function ensureDefaultsForOrganizations(organizations) {
+function ensureDefaultsForOrganizations(organizations = []) {
```

Comment:
This won't fix it if `null` is passed in, it only fixes it if the arg is missing, right? 

```js
function foo(foo = []) { console.log(foo) }

foo()
// []

foo(null)
// null
```

Reply:
ah `undefined` AND `null`. good catch